server/http_backend_test.go

```diff
@@ -110,25 +110,35 @@ func TestBackend_MaxConnections_BodyClose(t *testing.T) {
 		"/default",
 		"/default2",
 		"/ws",
+		"/proxy-seq",
+		"/proxy-seq-ref",
 	}
 
-	for _, p := range paths {
-		deadline, cancel := context.WithTimeout(context.Background(), time.Second*10)
+	t.Run("parallel", func(t *testing.T) {
+		for _, p := range paths {
+			p := p
+			t.Run("_"+p, func(st *testing.T) {
```

Comment:
```suggestion
			t.Run("_"+p, func(st *testing.T) {
```

Reply:
I added a description. Since we are running these funcs in parrallel `p` contains the last value from the loop slice, so all tests running against the last path. You could verify this via `println(p)` within the func.